src/api/app/controllers/staging/workflows_controller.rb

```diff
@@ -0,0 +1,47 @@
+class Staging::WorkflowsController < ApplicationController
+  before_action :require_login
+  before_action :set_project
+
+  def create
+    staging_workflow = @project.build_staging
+    authorize staging_workflow
+
+    staging_workflow.managers_group = Group.find_by(title: xml_hash)
+
+    unless staging_workflow.managers_group
```

Comment:
in the API we usually just throw exceptions for generic problems.

Reply:
We don't have to...

Reply:
Right, we can also duplicate code for no gain and increase code complexity.

Reply:
BTW: I forgot to flag this as nit-pick.

Reply:
I kept the same structure that other staging API have. I don't have a preference, but we should be consistent...

Reply:
it makes sense to change... the code is in the application controller to use it! :smile: